### ISSUE TITLE
missing exports condition warning logged in SvelteKit v2

### ISSUE THREAD

**Issue by THiragi** [PatrickG/html-svelte-parser#1]
Hi, I upgraded my SvelteKit app to v2 and see the following warning logged, so I would like to share it with you.

```bash
0:14:17 [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

html-svelte-parser@0.0.1

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
0:14:17 [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.
```

https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition

Thank you for creating this library.

**Comment by PatrickG** [comment]
Thank you for creating this issue.
Sorry for the late reply. Holidays.. :sweat_smile: 
I will publish a new version shortly.

**Comment by PatrickG** [comment]
I've just published v1.0.0 which fixes this.
